java/dataset/src/main/cpp/CMakeLists.txt

```
@@ -63,3 +63,6 @@ add_arrow_test(dataset_jni_test
                jni_util.cc
                EXTRA_INCLUDES
                ${JNI_INCLUDE_DIRS})
+               
+install(FILES jni_util.h
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/jni/dataset")
```

It looks like this fails the linter: https://github.com/apache/arrow/runs/7357633145?check_suite_focus=true#step:5:864

```suggestion
         
install(FILES jni_util.h DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/jni/dataset"
```

---

Thanks very much! I have fixed this as suggestion.